articles/azure-databricks/quickstart-create-databricks-workspace-portal.md

```diff
@@ -70,40 +70,39 @@ In this section, you create an Azure Databricks workspace using the Azure portal
     
-    Select **Create cluster**. Once the cluster is running, you can attach notebooks to the cluster and run Spark jobs. 
+    Select **Create cluster**. Once the cluster is running, you can attach notebooks to the cluster and run Spark jobs.
 
 For more information on creating clusters, see [Create a Spark cluster in Azure Databricks](https://docs.azuredatabricks.net/user-guide/clusters/create.html).
 
 
 ## Download a sample data file
 Download a sample JSON data file and save it into Azure blob storage.
 
-1. Download this sample JSON data file [from Github](https://raw.githubusercontent.com/Azure/usql/master/Examples/Samples/Data/json/radiowebsite/small_radio_json.json) onto your local computer. Right-click and save as to save the raw file locally. 
+1. Download this sample JSON data file [from Github](https://raw.githubusercontent.com/Azure/usql/master/Examples/Samples/Data/json/radiowebsite/small_radio_json.json) onto your local computer. Right-click and save as to save the raw file locally.
 
-2. If you don't already have a storage account, create one. 
-   - In the Azure portal, select **Create a resource**.  Select the **Storage** category, and select **Storage Accounts**  
-   - Provide a unique name for the storage account.
-   - Select **Account Kind**: **Blob Storage**
-   - Select a **Resource Group** name. Use the same resource group you created the Databricks workspace.
-   
-   For more information, see [Create an Azure Blob storage account](../storage/common/storage-quickstart-create-account.md). 
+2. If you don't already have a storage account, create one.
+    - In the Azure portal, select **Create a resource**. Select the **Storage** category, and select **Storage Accounts**
+    - Provide a unique name for the storage account.
+    - Select **Account Kind**: **Blob Storage**
+    - Select a **Resource Group** name. Use the same resource group you created the Databricks workspace.
+    
+    For more information, see [Create an Azure Blob storage account](../storage/common/storage-quickstart-create-account.md).
 
 3. Create a storage Container in the Blob Storage account and upload the sample json file into the container. You can use the Azure portal or the  [Microsoft Azure Storage Explorer](../vs-azure-tools-storage-manage-with-storage-explorer.md) to upload the file.
 
-   - Open the storage account in the Azure portal.
-   - Select **Blobs**.
-   - Select **+ Container** to create a new empty container.
-   - Provide a **Name** for the container, such as `databricks`. 
-   - Select  **Private (non anonymous access)** access level.
-   - Once the container is created, select the container name.
-   - Select the **Upload** button.
-   - On the **Files** page, select the **Folder icon** to browse and select the sample file `small_radio_json.json` for upload. 
-   - Select **Upload** to upload the file.
-   
-   
+    - Open the storage account in the Azure portal.
```

Comment:
We should use 3 spaces for indenting bullets. 4 spaces will render the bullets as code I believe.